Analyzers/include/AnalyzerCore.h

```diff
@@ -130,6 +132,14 @@ class AnalyzerCore: public SKFlatNtuple {
   //==== PU Reweight
   double GetPileUpWeight(int N_vtx, int syst);
 
+
+  //==== Btag setup
+  void SetupBTagger(std::vector<Jet::Tagger> taggers, std::vector<Jet::WP> wps, bool setup_systematics, bool period_dependant);
+  
+  //==== Is Btagged (using SF)
+  bool IsBTaggedCorrected(Jet j, Jet::Tagger tagger, Jet::WP WP, int systematic , bool applySF=true);
```

Comment:
Is there a reason this function is named as "IsBTaggedCorrected" not "IsBTagged"?
And I think it's better to have
(Jet j, Jet::Tagger tagger, Jet::WP WP, int systematic , bool applySF=true)
->
(Jet j, Jet::Tagger tagger, Jet::WP WP, bool applySF, int systematic)
I am trying not to use default argument in the analyzer, because users must know what they are doing. This is why I removed default pt and eta cuts for GetMuons() or etc (I just found SB used default arguments in GetPhotons, I will remove it soon.)

Reply:
I added two functions at first IsBTaggedCorrected and IsBTagged. Then I removed IsBTagged, adding applySF as input, but did not change the name. I changed it now to IsBTagged. And changed the arguments as suggested.